### PR TITLE
A set of Metasploit-exploitable vulnerabilities

### PR DESCRIPTION
**Pull request by todb-r7**
Hi CVE!

Here's a set of CVEs for some vulnerabilities that have Metasploit exploits associated with them, but no current CVE identifiers. I'd like to get all Metasploit exploits paired up with CVEs, but that means issuing CVEs for a bunch of weird little software applications, as well as contacting a few other CNAs for CVEs, and everything in between. 

Anyway, what do you think?

### CODE REVIEW

File: 2019/5xxx/CVE-2019-5618.json [hunk 1]
```diff
@@ -1,18 +1,81 @@
-{
```
Comment (cve-team) [inline 2019/5xxx/CVE-2019-5618.json:1]: A language tag was used multiple times: problemtype value #0

File: 2019/5xxx/CVE-2019-5619.json [hunk 2]
```diff
@@ -1,18 +1,81 @@
-{
```
Comment (cve-team) [inline 2019/5xxx/CVE-2019-5619.json:1]: A language tag was used multiple times: problemtype value #0

File: 2019/5xxx/CVE-2019-5620.json [hunk 3]
```diff
@@ -1,18 +1,81 @@
-{
```
Comment (cve-team) [inline 2019/5xxx/CVE-2019-5620.json:1]: A language tag was used multiple times: problemtype value #0

File: 2019/5xxx/CVE-2019-5621.json [hunk 4]
```diff
@@ -1,18 +1,81 @@
-{
```
Comment (cve-team) [inline 2019/5xxx/CVE-2019-5621.json:1]: A language tag was used multiple times: problemtype value #0

File: 2019/5xxx/CVE-2019-5622.json [hunk 5]
```diff
@@ -1,18 +1,81 @@
-{
```
Comment (cve-team) [inline 2019/5xxx/CVE-2019-5622.json:1]: A language tag was used multiple times: problemtype value #0

File: 2019/5xxx/CVE-2019-5623.json [hunk 6]
```diff
@@ -1,18 +1,81 @@
-{
```
Comment (cve-team) [inline 2019/5xxx/CVE-2019-5623.json:1]: A language tag was used multiple times: problemtype value #0

**Comment by todb-r7** [comment]
oh no

**Comment by todb-r7** [comment]
ah yes this was validated a year ago is why. One sec.

**Comment by todb-r7** [comment]
Fixed in the multiple problemtype error in https://github.com/CVEProject/cvelist/pull/3697/commits/2582416f012218b4ec18ced78907450812efe7cb

**Pull request merged**